### ISSUE TITLE
[bench request]: Add TDE general v0.1 (tdelab, ModernBERT-base 150M, native probabilities, in-process adapter in PR #38)

### ISSUE THREAD

**Issue by e13ven-arch** [fstandhartinger/jevbench#39]
Request to benchmark **TDE general v0.1** (tdelab; ModernBERT-base encoder, 150,208,513 parameters, pointer readout, native probabilities) on the complete frozen JevBench v1.2 set. Adapter and registration are in PR #38.

## System details
- **Name:** tde-general-v0.1
- **Code:** https://github.com/e13ven-arch/tde — tag `v0.1.0-rc1` (commit `5e17ce9`); package `tde`, Apache-2.0
- **Weights:** `tdelab/tde-general-v0.1`, hub revision `a8a0eb0f6ac21a8d95d1a135df090f3b1b1da93e` (Apache-2.0). Self-contained folder: `config.json`, `tokenizer/`, `best.pt` (state dict) / `model.safetensors`; SHA-256 of every file in `MANIFEST.json`.
- **Architecture:** one bidirectional forward pass over `state + question + candidates`; each candidate is prefixed by a `[MASK]` marker, its representation is the marker state plus the mean over its tokens, scored by a dot product against a final `[MASK]` state, softmax over the exact label set. No text is generated; **0 output tokens**. Backbone initialised from the GLiClass modern-base v3.0 encoder, then fully fine-tuned.
- **Readout mapping:** choice criteria map → candidates `name: description` in the task's label order; score criteria list → ordered levels with a learned level-index embedding, expected level as score; noul → yes/no candidates carrying the true/false criteria text. Probabilities are used as trained (fitted post-hoc temperatures are 0.98–1.05, so none is applied).
- **Training data:** public classification / NLI / QA sets (banking77, clinc150, AG News, SST-5, GoEmotions, BoolQ, SNLI, LegalBench rule tasks, HotpotQA), the LocalLLaMA/typed-decisions train split, and rule-generated policy / multi-hop / temporal worlds with programmatic labels (generator in the repo). **No JevBench item, public or held out, and no paraphrase of one was used; no output of Jev was used for training, tuning or selection.** A 13-gram overlap check between every training state and the 231 public items found 0 overlaps (`scripts/contamination_check.py`).

## How to run (in-process, no server)
```bash
pip install "git+https://github.com/e13ven-arch/tde@v0.1.0-rc1"
# adapter + CLI registration: PR #38 (jevbench/adapters/tde_local.py, docs/v1.2-additions-tde.md)
SNAP=$(python -c "from huggingface_hub import snapshot_download;print(snapshot_download('tdelab/tde-general-v0.1', revision='a8a0eb0f6ac21a8d95d1a135df090f3b1b1da93e'))")
JEVBENCH_WARM_LOAD=1 python -m jevbench.cli run --tasks <public or held-out jsonl> --adapter tde_local \
  --endpoint "$SNAP" --model tde-general-v0.1 --results results/tde/results.jsonl --raw-dir /tmp/tde-raw \
  --ledger results/tde/ledger.jsonl --cap-usd 1 --reserve-usd 0
```
CPU works (≈0.1 s/decision); CUDA or Apple MPS is faster. Memory < 2 GB. Inputs longer than the model's budget are truncated on the state side (1,024-token sequence budget by default; `max_total` can be raised to 8,192).

## Public self-run
Harness revision `a1799db9673bf59010bc698753e729b04132431f`, dataset hash `dc3995d8ae1e2fc8e81ce38431add509eb8bb39b85aadfd0c7c32079382dde51`, all 231 public items through `jevbench.runner.Runner` + `jevbench.summarize` with the adapter above (Apple M5 Pro, MPS, batch size 1).

| easy | standard (public original) | hard (public) | overall | Brier | ECE (10 bins) | ordinal MAE |
|---:|---:|---:|---:|---:|---:|---:|
| 48/48 (100%) | 44/72 (61.1%) | 38/111 (34.2%) | 130/231 (56.3%) | 0.552 | 0.115 | 0.644 |

231/231 valid, strict-valid, 0 renormalised. Latency p50 0.0165 s, p95 0.060 s (local, serial, after warm load; not your server's numbers).

## Disclosure
- The model was selected on our own calibration split and on the typed-decisions test set, never on JevBench items; the public-item numbers above are a single post-hoc run.
- Known weak spots on the hard tier: multi_hop, trap, temporal_numeric (near chance). Ordinal questions are the least calibrated primitive.
- A second checkpoint (`tde-typed-specialist-v0.1`, same model + typed-decisions specialist stage) will be published separately; it is not the one requested here.

**Comment by fstandhartinger** [comment]
Thanks — this is logged for reachability checks and the JevBench measurement queue. We’ll only make benchmark claims after our own run.

- Harold

**Comment by fstandhartinger** [comment]
Thanks for offering a benchmark-scoped endpoint. A full frozen run includes held-out items, and an endpoint you operate would receive their state and question text. Could you provide open weights/a self-hostable build or an endpoint hosted on maintainer-controlled infrastructure? If neither is available, please confirm explicitly that you want us to send the full frozen set, including held-out items, to your endpoint; we would document that condition clearly. Please do not post credentials here. The current label-only `/score` route receives no Calibration credit unless a native probability distribution is frozen before measurement.

**Comment by e13ven-arch** [comment]
To clarify, since the note above mentions an endpoint and a `/score` route: this submission has **no endpoint operated by us**. It is open weights plus a self-hostable, in-process build:

- Weights: `tdelab/tde-general-v0.1` on Hugging Face, Apache-2.0, pinned revision `a8a0eb0f6ac21a8d95d1a135df090f3b1b1da93e` (self-contained folder; SHA-256 of every file in `MANIFEST.json`).
- Code: `pip install "git+https://github.com/e13ven-arch/tde@v0.1.0-rc1"`, Apache-2.0.
- Adapter: in-process `tde_local` (PR #38), so the held-out items never leave your machine. It returns a **native probability distribution over the exact declared label set for every question type**, passed to your scorer unmodified (no temperature, no renormalisation, no label filling; anything else is reported as an adapter error). 0 output tokens.

So the full frozen set, including held-out items, can be run entirely on maintainer-controlled infrastructure with the command in the issue body (`--reserve-usd 0` matters, otherwise the ledger skips local runs). Happy to adjust anything else needed for the queue.
